README.md

```diff
@@ -49,6 +49,22 @@ http:
   # ssl: enables HTTPS protocol if set to `true` (optional).
   ssl: false
 
+# provider: the provider that topograph will use unless otherwise specified by the topology request.
+# Valid options include "aws", "oci", "gcp", "cw", "baremetal" or "test".
+# Default value if not specified with be "test"
+provider: "aws"
+
+# engine: the engine that topograph will use unless otherwise specified by the topology request.
+# Valid options include "slurm", "k8s", or "test".
+# Default value if not specified will be "test".
```

Comment:
I think we don't have default value anymore. If the value is not provided here, and is missed in the payload, we are throwing an error, correct?

Reply:
Default values aren't necessary anymore.